src/In.ProjectEKA.OtpService/Otp/OtpSender.cs

```diff
@@ -1,32 +1,56 @@
 namespace In.ProjectEKA.OtpService.Otp
 {
+    using System;
     using System.Threading.Tasks;
     using Clients;
     using Common;
+    using Microsoft.Extensions.Configuration;
 
     public class OtpSender : IOtpSender
     {
         private readonly IOtpRepository otpRepository;
         private readonly IOtpGenerator otpGenerator;
         private readonly ISmsClient smsClient;
+        private readonly OtpProperties otpProperties;
 
-        public OtpSender(IOtpRepository otpRepository, IOtpGenerator otpGenerator, ISmsClient smsClient)
+        public OtpSender(IOtpRepository otpRepository, IOtpGenerator otpGenerator, ISmsClient smsClient, OtpProperties otpProperties)
         {
             this.otpRepository = otpRepository;
             this.otpGenerator = otpGenerator;
             this.smsClient = smsClient;
+            this.otpProperties = otpProperties;
         }
 
         public async Task<Response> GenerateOtp(OtpGenerationRequest otpGenerationRequest)
         {
-            var otpValue = otpGenerator.GenerateOtp();
-            var sendOtp = await smsClient.Send(otpGenerationRequest.Communication.Value, otpValue);
+            var generateOtp = otpGenerator.GenerateOtp();
```

Comment:
can be renamed to otp instead of generateOtp

Reply:
done